.github/workflows/unit-tests.yml

```
@@ -0,0 +1,105 @@
+name: Unit Tests
+
+on:
+  pull_request:
+  push:
+    branches:
+    - main
```

```suggestion
    - develop
```

Our `main` branch isn't really used atm.

---

ah right thanks for catching that